<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifyTable2</class>
 <widget class="QDialog" name="DialogModifyTable2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>278</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>WenQuanYi Micro Hei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>修改项目</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语音识别内容</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem0">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btModifyConfirm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定写入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btQuit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>89</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语音合成内容</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>129</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>录音更新</string>
   </property>
   <widget class="QPushButton" name="btRecordStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btRecordFinish">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>录音完成</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择更新录音</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
